src/js/tech/html5.js

```diff
@@ -321,7 +321,7 @@ class Html5 extends Tech {
     if ('webkitDisplayingFullscreen' in video) {
       this.one('webkitbeginfullscreen', function() {
         this.one('webkitendfullscreen', function() {
-          this.trigger('fullscreenchange');
+          this.trigger('fullscreenchange', 'not-fullscreen');
```

Comment:
I am seeing two `fullscreenchange` events on the player currently on ios.
I'm not sure whether it's better to keep the `isFS` info on the player or whether it should go into techs. I guess since the player div is the one that goes into fullscreen, it kind of makes for it to live on the player and ios would just need to update the player of this. In which case, should we just make the in the `handleTechFullscreenChange` to be `data && this.isFullscreen(data.fullscreen);`?